src/ListPage.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import PokeyList from './PokeyList.js';
+import SearchPokemon from './SearchPokemon.js';
+import Sort from './Sort.js';
+import fetch from 'superagent';
+
+export default class App extends Component {
+  state = {
+    filter: '',
+    textString: '',
+    changeOne: 'ascending',
+    changeTwo: 'pokemon',
+    pokemonData: []
+  }
+
+  fetchPokemon = async () => {
+    const response = await fetch.get(`https://alchemy-pokedex.herokuapp.com/api/pokedex?pokemon=${this.state.filter}&perPage=1000`);
+    this.setState({ pokemonData: response.body.results });
+  }
+
+  componentDidMount = async () => {
+    this.fetchPokemon();
+  }
+
+  buttonClick = async () => {
+    this.fetchPokemon();
+  }
+
+  textChange = (e) => {
+    this.setState({
+      filter: e.target.value
+    })
+  }
+
+  changeOne = (e) => {
```

Comment:
your change handlers will need to take different type of fetches in order to sort and filter directly from the api and will need to pay attention on how to use the state you had in your app previously.